public/ge0.html

```diff
@@ -104,13 +131,20 @@
     const lon = ${ lon };
     const zoom = ${ zoom };
     const map = L.map('map').setView([lat, lon], zoom);
```

Comment:
Can you please also replace all `const` and `let` to `var`? Looks like some Android 5 devices may not work properly.

Reply:
fixed